### DISCUSSION TITLE
Payload V3 - Updated templates

### DISCUSSION THREAD

**Discussion by Nathan-Nesbitt** [payloadcms/payload#6792] [Q&A]
Hey first off, great job on the move from V2 to V3. I really like the new design!

I wanted to see if there is an intention to upgrade the three existing templates from V2 into V3. If there is, is there any timeline for it?

https://github.com/payloadcms/payload/tree/main/templates

Looks like the V3 tagged branches have some files, but I don't think any of these templates have been moved yet as they all appear to follow the V2 pattern:

https://github.com/payloadcms/payload/tree/v3.0.0-beta.47/templates

Just wanted to check before I build all the functionality they have into V3 by myself! 

Thanks!

**Comment by paulpopus** [comment]
I commented in another thread about this but in cause anyone is coming across:

The website template has been ugpraded already (although overdue an update, soon!) to v3

```
npx create-payload-app@beta --use-pnpm -t website
```

The ecommerce template work has been started although put on ice right now as we work through the issue queue and towards stability. It will have a lot more features than before!

**Reply by grantspilsbury** [reply]
Thanks @paulpopus. Do you know when the ecomm template will be updated to use v3?

**Reply by arfa123** [reply]
any update on ecommerce template for v3?

**Reply by denizbinay** [reply]
A version with more features sounds fantastic! However, if that means no e-commerce template will be available for a while, that would be inconvenient. How about doing a quick port for v3, similar to the website template, and addressing the full overhaul later?

**Reply by paulpopus** [reply]
It doesn't function that way unfortunately since it's all fresh. Anyway the bulk of it was done back in July, so I'll aim to get this one out in the next several weeks

**Reply by denizbinay** [reply]
It’s great to hear that so much progress has already been made. Looking forward to seeing it in action! 🙌

**Comment by joaogsleite** [comment]
Do you know if the e-commerce template will be ready for the payload 3 release? Thank you!

**Reply by junkmasterjim** [reply]
gentle boost for this. wondering the same

**Reply by paulpopus** [reply]
It's gonna be back on my plate now that 3.0 is out

**Reply by mrwebninja** [reply]
@paulpopus any update on when the payload 3 e-commerce template will be ready?

**Reply by Will-Bill** [reply]
@paulpopus i can see a lot of work being done on the ecommerce feat. We are about to embark on an ecommerce project using Payload. Can I ask if there are any plans as to when there may be a beta for Payload eCommerce template?

**Reply by DanRibbens** [reply]
@Will-Bill we have some more improvements coming before we release the beta.
If I were you I would start with the template even in it's draft state because it does have a lot of really good functionality already and will be a better start than starting from blank. When we do release it I would diff between your current state and the new release to see if any of the final changes are useful for you.

Here is a link to the latest template source: https://github.com/payloadcms/payload/tree/feat/ecommerce-template/templates/ecommerce

**Reply by Will-Bill** [reply]
@DanRibbens thanks - this was my thinking, to use it anyway and patch any changes later. Our launch is not until Jan/Feb 26 - do you have any indication if the template will be released before then? Of course wouldn't hold any dates given as gospel, would just be good to get an understanding as to your timeline.

**Comment by tomtana** [comment]
#8297 I think this is the pr for it. @paulpopus thanks for driving this 🙏

**Comment by mvarchdev** [comment]
Hey guys! What is the status of ecommerce? We need to build ecommerce app for our customer and decided to try payloadcms for this. Can we already try to use it?

**Reply by TraVv1S** [reply]
I'm really looking forward to it!

**Reply by sanjayguwaju** [reply]
I'm also looking forward it.

**Comment by mnishanth02** [comment]
is e-commerce temaplte for V3 is out or any tentative dates ?

**Comment by dynjo** [comment]
👍🏼 for the ecommerce template, any news?

**Reply by paulpopus** [reply]
Work in progress, design is done, new UI for variants:
![image](https://github.com/user-attachments/assets/c1ba49ad-f41a-400f-96b5-704a97f585cc)

Will have the initial working version ready very soon (like the next few days) but it will likely stay a bit longer in QA, [track progress here](https://github.com/payloadcms/payload/pull/8297). This is a very high priority for us
